tests/content/test_content_tasks.py

```diff
@@ -0,0 +1,19 @@
+from bulbs.content.models import FeatureType
+from bulbs.content.tasks import update_feature_type_rates
+from bulbs.utils.test import BaseIndexableTestCase
+from bulbs.contributions.models import ContributorRole, FeatureTypeRate
+
+
+class FeatureTypeTasksTestCase(BaseIndexableTestCase):
+    def test_update_feature_type_rates(self):
+        role = ContributorRole.objects.create(name='Test Role', payment_type=1)
+        feature_type = FeatureType.objects.create(
+            name='Test Feature Type',
+            slug='test-feature-type',
+            instant_article=False)
```

Comment:
I'll adhere to whatever the prevailing style is here. I assumed that since python has no end of block syntax that indentation would be sufficient, so I used lisp style.